tests/foreman/rhai/test_rhai.py

```diff
@@ -125,9 +124,10 @@ def test_unregister_system_from_rhai(self):
                 Navigator(self.browser).go_to_insights_systems()
                 # Click on the unregister icon 'X' in the table against the
                 # registered system listed.
-                session.nav.click(
-                    locators['insights.unregister_system']
-                )
+                strategy, value = locators['insights.unregister_system']
+                session.nav.click((strategy, value % self.rhai.hostname),
```

Comment:
This will make easier to read:

```python
session.nav.click(
    (strategy, value % self.rhai.hostname),
    wait_for_ajax=True,
    timeout=40
)
```